workers/advisor/build.gradle.kts

```diff
@@ -40,6 +40,18 @@ tasks.withType<JibTask> {
     notCompatibleWithConfigurationCache("https://github.com/GoogleContainerTools/jib/issues/3132")
 }
 
+repositories {
+    exclusiveContent {
+        forRepository {
+            maven("https://repo.gradle.org/gradle/libs-releases/")
+        }
+
+        filter {
+            includeGroup("org.gradle")
+        }
+    }
+}
```

Comment:
You could add this to a new convention plugin in `buildSrc` to avoid having to repeat the config in every worker (e.g., `ort-server-worker-conventions.gradle.kts`).

Reply:
I am not sure how to proceed here. If the approach from #3330 was used, these duplicate declarations would not be needed. So, there are multiple options:
* live with the duplication for now and deal with it later by either removing it (if it is not needed) or move it to a plugin
* move it to a plugin now which may become obsolete in the future
* decide first on an approach to simplify access to plugin information and postpone this PR